app/src/main/scala/org/alephium/explorer/api/model/Pagination.scala

```diff
@@ -23,7 +23,7 @@ object Pagination {
   val defaultPage: Int  = 1
   val defaultLimit: Int = 20
   val maxLimit: Int     = 100
-  val `100K`: Int     = 100_000
+  val `100K`: Int       = 100000
```

Comment:
ah, any reason you went back to no `_`?

Reply:
scalafmt didn't like it: `error: Invalid literal number`

Reply:
oh ok, I'll check if we can change that, I kinda like this format